### ISSUE TITLE
make vm fails on a fresh Mac Pro Laptop

### ISSUE THREAD

**Issue by malachig** [genome/gms#104]
I just did a `make vm` on a laptop that has never seen the sGMS before.  The installation of virtualbox and vagrant seemed to go smoothly.  But when it goes to add the first VM image, I get this:

```
make vm
#
# done-host/vmaddbox:
#
# intializing vagrant (VirtualBox) VM...
# these steps occur before the repo is visible on the VM
#
(vagrant box list | grep '^precise64' >/dev/null && echo "found vagrant precise64 box") || (echo "installing vagrant precise64 box" && vagrant box add precise64 http://files.vagrantup.com/precise64.box)
installing vagrant precise64 box
Downloading box from URL: http://files.vagrantup.com/precise64.box
Download failed. Will try another box URL if there is one.-)
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.

Failed writing body (0 != 869)
make: *** [done-host/vmaddbox] Error 1
```

The URL is valid and I can download the precise64.box manually.  I suspect some kind of permissions issue with the way vagrant is being set up...

Similar to this issue here:
https://github.com/kitchenplan/chef-applications/issues/98

For some reason, my ~/.vagrant.d is created with root as the owner...

**Comment by malachig** [comment]
Adding this line to the make right before attempting the `vagrant box add` seems to have cleared up this issue: `sudo chown -R `whoami`: ~/.vagrant.d`

Full make target:

``` make
   178 done-host/vmaddbox: done-host/vminstall
   179   #
   180   # $@:
   181   #
   182   # intializing vagrant (VirtualBox) VM...
   183   # these steps occur before the repo is visible on the VM
   184   #
   185   sudo chown -R `whoami`: ~/.vagrant.d
   186   (vagrant box list | grep '^precise64' >/dev/null && echo "found vagrant precise64 box") || (echo "installing vagrant precise64 box" && vagrant box add preci§
   187   touch $@
```

**Comment by sakoht** [comment]
Yeah I guess the first run of vagrant makes that dir, and b/c it is
happening under sudo on a new box, it gets made with root permissions.

**Comment by malachig** [comment]
I encountered a similar issue with the gms/.vagrant dir having the wrong permissions.  Fixed that, made a couple other minor tweaks and tested both `make vm` and 'make vminit && vagrant ssh && make' on both my laptop and mac pro desktop.  Seems to be working smoothly.  Relevant commits:

https://github.com/genome/gms/commit/2f2faebc928aae8d1a51604ec0baaa02dfdd235f
https://github.com/genome/gms/commit/7c4e8b5c9b7fe4ca7cd74175783a54e64dcd0811

Closing for now.
